### ISSUE TITLE
[Feature Request] Use mocha installed by user (in node_modules) instead of have it as the extension dependency

### ISSUE THREAD

**Issue by ris58h** [maty21/mocha-sidebar#131]
Is it possible?

**Comment by maty21** [comment]
Yep there is an option to set mocha path

U should set path under mocha.path option more information can be found in
the README section

**Comment by maty21** [comment]
I don't think it should removed since it making easier in case you don't have micha installed locally.
 Maybe we should think of changing the order .
First look for local mocha installation and then as a fallback to you the one that installed in the extension how is it   sounds ?
